java/core/src/main/java/com/google/protobuf/ByteString.java

```diff
@@ -1603,6 +1606,7 @@ protected int getOffsetIntoBytes() {
   // Keep this class private to avoid deadlocks in classloading across threads as ByteString's
   // static initializer loads LiteralByteString and another thread loads BoundedByteString.
   private static final class BoundedByteString extends LiteralByteString {
+    private static final long serialVersionUID = 1L;
```

Comment:
This is a duplicate definition. I didn't notice in the earlier review that this was already defined below in the Serializable section.

Reply:
Ooops! Sorry, I missed that. Fixed.